examples/hikari_lightbulb_basic_queue/main.py

```diff
@@ -1,38 +1,49 @@
 import os
-
-from consts import TOKEN, PREFIX
+from typing import Any
 
 import hikari
 import lightbulb
+from consts import PREFIX
+from consts import TOKEN
+
 import lavasnek_rs
 
 
 class Data:
     """Global data shared across the entire bot, used to store dashboard values."""
+
     def __init__(self) -> None:
         self.lavalink: lavasnek_rs.Lavalink = None
 
 
 class MusicBot(lightbulb.Bot):
     """Subclass of lightbulb's Bot object, used to store the lavalink client."""
-    def __init__(self, *args, **kwargs):
+
+    def __init__(self, *args: Any, **kwargs: Any):
         super().__init__(*args, **kwargs)
         self.data = Data()
 
+
 # You may want to enable ALL intents here
 bot = MusicBot(token=TOKEN, prefix=PREFIX)
 
 
 @bot.listen(hikari.StartingEvent)
-async def starting_load_extensions(event):
+async def starting_load_extensions(_: hikari.StartingEvent) -> None:
```

Comment:
With the addition of typehints, having the event in the listen is redundant

Reply:
so the event should just be removed from @bot.listen?